JS-112_javascript-arrays/loop-through-arrays/exercise/search-an-array/js/search-list.js

```diff
@@ -0,0 +1,11 @@
+const inStock = ['pizza', 'cookies', 'eggs', 'apples', 'milk', 'cheese', 'bread', 'lettuce', 'carrots', 'broccoli', 'potatoes', 'crackers', 'onions', 'tofu', 'limes', 'cucumbers'];
+const search = prompt('Search for a product.');
+let message;
+
+if( inStock.includes(search) ) {
```

Comment:
No issue here, just a note. Be consistent in writing code blocks. The spaces after the `if` or `for` keywords sometimes have space after them and sometimes they do not. I would recommend installing ESLint on your VS Code to check for these type of inconsistencies.